app/models/mixins/miq_provision_mixin.rb

```diff
@@ -55,7 +55,7 @@ def get_owner
     @owner ||= begin
       email = get_option(:owner_email).try(:downcase)
       return if email.blank?
-      User.find_by_lower_email(email, get_user).tap do |owner|
+      User.lookup_by_lower_email(email, get_user).tap do |owner|
```

Comment:
I love the name of `find_by_lower_email` rather than `find_by_email` and the developer has to know that it is case-insensitive.

(Not asking for a change. Just making an observation)

Reply:
better yet, would prefer not having this method at all and have:

```ruby
find_by(:email => INSENSITIVE(get_user))
```

but that is out of scope here - don't have that implemented yet